From e621c74f96c7f4093829d7a4b91f91c589bb3949 Mon Sep 17 00:00:00 2001
From: Christopher Pelloux <git@chp.io>
Date: Thu, 4 Jun 2020 21:11:26 -0400
Subject: [PATCH 1/1] package/json-c: bump version to 0.14

Notes:

- json-c now uses cmake instead of autoconf
- This version also brings support to the much welcomed feature for
  parsing uint64_t types

Signed-off-by: Christopher Pelloux <git@chp.io>
---
 package/json-c/json-c.hash |  2 +-
 package/json-c/json-c.mk   | 14 +++-----------
 2 files changed, 4 insertions(+), 12 deletions(-)

diff --git a/package/json-c/json-c.hash b/package/json-c/json-c.hash
index 0e03007476..93eaff67c3 100644
--- a/package/json-c/json-c.hash
+++ b/package/json-c/json-c.hash
@@ -1,4 +1,4 @@
 # From https://github.com/json-c/json-c/wiki
-sha256 b87e608d4d3f7bfdd36ef78d56d53c74e66ab278d318b71e6002a369d36f4873  json-c-0.13.1.tar.gz
+sha256 b377de08c9b23ca3b37d9a9828107dff1de5ce208ff4ebb35005a794f30c6870  json-c-0.14.tar.gz
 # Locally calculated
 sha256 74c1e6ca5eba76b54d0ad00d4815c8315c1b3bc45ff99de61d103dc92486284c  COPYING
diff --git a/package/json-c/json-c.mk b/package/json-c/json-c.mk
index 2788fe563b..3e17effdad 100644
--- a/package/json-c/json-c.mk
+++ b/package/json-c/json-c.mk
@@ -4,19 +4,11 @@
 #
 ################################################################################
 
-JSON_C_VERSION = 0.13.1
+JSON_C_VERSION = 0.14
 JSON_C_SITE = https://s3.amazonaws.com/json-c_releases/releases
 JSON_C_INSTALL_STAGING = YES
 JSON_C_LICENSE = MIT
 JSON_C_LICENSE_FILES = COPYING
 
-# update config.h.in timestamp to avoid autoheader run
-define JSON_C_UPDATE_CONFIG_TIMESTAMP
-	touch $(@D)/config.h.in
-endef
-
-JSON_C_POST_EXTRACT_HOOKS += JSON_C_UPDATE_CONFIG_TIMESTAMP
-HOST_JSON_C_POST_EXTRACT_HOOKS += JSON_C_UPDATE_CONFIG_TIMESTAMP
-
-$(eval $(autotools-package))
-$(eval $(host-autotools-package))
+$(eval $(cmake-package))
+$(eval $(host-cmake-package))
-- 
2.17.1

